testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/MultinodeHiveCaching.java

```diff
@@ -72,7 +72,7 @@ public void extendEnvironment(Environment.Builder builder)
     {
         builder.configureContainer(COORDINATOR, container -> container
                 .withCopyFileToContainer(forHostPath(dockerFiles.getDockerFilesHostPath("conf/environment/multinode/multinode-master-jvm.config")), CONTAINER_PRESTO_JVM_CONFIG)
```

Comment:
while you are here can use `io.trino.tests.product.launcher.docker.DockerFiles.ResourceProvider` here. It is cleaner and safer.